docs/tools/locust/api-parsers.md

```diff
@@ -0,0 +1,167 @@
+---
+sidebar_position: 11
+---
+# API Parsers
```

Comment:
I will say there has been confusion in the past between the Prance parser being able to parse the swagger file versus the Spectral linter finding it valid.  I don't know if that's the place to call this out but I think it's easy to think they are one in the same.

Reply:
Interesting @duncan-truss so you're saying that the Spectral linter and the Prance parser don't agree on the validity of our Swagger files? Do you know of an example where this happens? It makes sense to be explicit about this difference so folks aren't confused by the discrepancy.

Reply:
Yea pre-commit is setup to lint the prime and support APIs according to the spectral linter with some custom rules thrown in https://github.com/transcom/mymove/blob/master/swagger-def/.spectral.yml.  The internal and ghc APIs are not up to the style guide enforced here for things like paths being kebab-case or body properties being camelCase because of there being legacy code that requires client side changes if the API shape changes.

Another pre-commit step to separately parse all of the APIs using Prance would actually be a nice way to know if we've broken the load test parser.  Load tests could be broken if not updated regardless if the API is parseble though for other reasons but it would be a start.

Reply:
Ah good point. I can add a comment about this.